### ISSUE TITLE
Add next board meeting date to DSF minutes

### ISSUE THREAD

**Issue by nanorepublica** [django/djangoproject.com#1705]
Allow the DSF minutes to include the next board date (when known) to the minutes.

This improves transparency to the wider community.

**Comment by bmispelon** [comment]
Same question as #1704 : does this change need to be approved by the DSF board first, or can someone start writing a PR already?

**Comment by nanorepublica** [comment]
I asked Thibaud in a DM as I wasn't aware the meeting minutes were a model. He gave a thumbs up to both of these changes.

**Comment by thibaudcolas** [comment]
Yes please! This isn’t significant enough to require a board vote.

If anyone wants to proceed with this, please make sure to:

- Make this field optional
- Choose wording to make it clear the date is provisional / can change (we often have special meetings being called, or reschedule just to have more board members available)

**Comment by thibaudcolas** [comment]
So feedback from the board is good, with a suggestion that the exact date could also be replaced or complemented with a generic note that "meetings are generally the second Thursday of the month but are occasionally rescheduled for attendance".

---

So, I think we could add this at the end of the template that displays the minutes:

```
<p>Board meetings are generally the second Thursday of the month but are occasionally rescheduled for attendance.{% if next_meeting_date %} Next meeting scheduled for: {{ next_meeting_date }}{% endif %}</p>
```

With `next_meeting_date`set up as either a date or plain-text field.

**Comment by thibaudcolas** [comment]
cc our 2025 Secretary @knyghty

**Comment by FarhanAliRaza** [comment]
The next_meeting_date should be part of the `Meeting` model ?

**Comment by sarahboyce** [comment]
Given the DSF meeting minutes now live in https://github.com/django/dsf-minutes (see https://github.com/django/djangoproject.com/issues/2079). I suppose we can close this issue and this feedback can be incorporated in the DSF secretary monute writing process (cc @knyghty)

**Comment by knyghty** [comment]
Yes, and I would say it's a little tricky to update since each year the incoming board chooses a new meeting time to make sure it's as convenient as possible for everyone.

**Comment by adamzap** [comment]
Thanks for the insight. I'll close this!
